spec/models/degree_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe Degree, type: :model do
+  it { is_expected.to validate_presence_of(:type_of_degree) }
+  it { is_expected.to validate_presence_of(:subject) }
+  it { is_expected.to validate_presence_of(:institution) }
+  it { is_expected.to validate_presence_of(:class_of_degree) }
+  it { is_expected.to validate_presence_of(:year) }
```

Comment:
Should we start doing more validation than just presence, or leave that until later? (cc @duncanjbrown / @paulrobertlloyd)

Reply:
Later but **very soon**, after this PR let's create some cards describing the kind of validations we want to apply.

Reply:
What's the case for not doing it right now? It feels like it's part of this feature — we include `_form_error_summary`, for example.

Reply:
Notably, `_form_error_summary` can't show multiple error messages right now, it just plucks out the `.first`. 

There's a new version of the [form builder gem](https://github.com/DFE-Digital/govuk_design_system_formbuilder/) we're using, perhaps we should work on bringing that it in a separate PR and then addressing the validations up until now as its own piece?

I'm not sure what is best here.

Reply:
How about this: https://trello.com/c/GStGMZPg/661-define-validation-criteria-for-fields-in-all-the-form-pages-weve-built-so-far-in-the-app

Hopefully dependabot will do the form builder gem update for us?

Reply:
@duncanjbrown IMO we should not add other validations now to avoid adding discussions on that topic in this PR, and postponing the merging. Let's add all those many validations in a different PR.

Reply:
@kjdchapman yes it is showing just the first error, but not sure if it is a bad thing, we might do not want to overwhelm the user with many errors for any fields.
That's a good question for @paulrobertlloyd